Makefile

```diff
@@ -227,7 +225,7 @@ CONFIG_FILE = $(CONFIG_FILE_PREFIX)$(CONFIG_HASH)
 mismatched_config_file_check: | $(BUILD_DIR)/.
 	$(BRIEF_PARTIAL) Checking for mismatched config...
 	$(COMMAND) ls $(CONFIG_FILE_PREFIX)* 2>/dev/null | grep -v $(CONFIG_FILE) | xargs -ri sh -c 'echo "Mismatched config file \"{}\" detected.  You need to \"make clean\"."; false'
-	$(BRIEF) No mismatched config found
+	$(BRIEF) No mismatched config found. Nothing to build.
```

Comment:
This target still runs as part of other targets which may still build stuff, right? In that case, I wouldn't add this message.

Reply:
You are right ... I discovered that for some usages, this extra message is -actually- wrong and misleading.

I seem to have taken it out in my branch. This diff-block is possibly stale.